src/cellrank/pl/_circular_projection.py

```diff
@@ -69,13 +69,15 @@ def _get_optimal_order(data: Lineage, metric: Metric_T) -> Tuple[float, np.ndarr
 
 
 @d.dedent
-@deprecated_arg_names({"labeldistance": "label_distance", "labelrot": "label_rot"})
+@renamed_arg('labeldistance', 'label_distance')
```

Comment:
Let's just remove this, it's been deprecated for very long!

Reply:
Got it!

Reply:
Thanks a lot!